### DISCUSSION TITLE
Comments section in "Location" tab ?

### DISCUSSION THREAD

**Discussion by retho-p** [netbox-community/netbox#9787] [Ideas]
Hello, 

I wanted to know if there was a reason for not having a "Comments" section in the "Location" tab as there can be for the "Site" or "Racks" tab.
![image](https://user-images.githubusercontent.com/63599476/179920136-49270233-5aea-4ea9-b2cb-a1d448f3da8e.png)

Would it be useful for netbox users? 

For my case yes, it would allow me to put indications such as the link to the documentation / how to get to the room (if "Location" is modeled for a room for example).

What do you think about it ?

**Comment by candlerb** [comment]
I agree it's a bit inconsistent. With v3.2.6 I see:
* Location has "Description" but not "Comments"
* Rack has "Comments" but not "Description"
* Site has both "Description" and "Comments"
